### ISSUE TITLE
Build tvOS Target error

### ISSUE THREAD

**Issue by Nien97788** [fkuehne/upnpx#88]
I build project with tvos target. I got unpnx.framework
I copied to my project and added to framework, when i build, i got error message
upnpx/src/api/BasicUPnPService.h:40:9: 'UPnPServiceConstants.h' file not found
I use swift language

**Comment by congzing** [comment]
Hey how can you resolve this problem? I got same thing

**Comment by tifroz** [comment]
I observed the same with a Swift iOS project.

I fixed it by changing the `TargetMembership` of the flagged `.h` files from `Project` to `Public`in the upnpx framework.

1. Open the upnpx project
2. Under **/sources/api/** change the target membership to `Public` for `UPnPServiceConstants.h` , `UPnPServiceConstantsAVTransport1.h`, `UPnPServiceConstantsRenderingControl1.h`, `UPnPServiceConstantsConnectionManager1.h`, `UPnPServiceConstantsContentDirectory1.h`
3. Build and use the resulting `upnpx.framework` in your project

<img width="1386" alt="screen shot 2018-12-06 at 12 12 09 pm" src="https://user-images.githubusercontent.com/150367/49609749-4c259a00-f951-11e8-8b84-21ee9348ae6e.png">
